curlylint/parse.py

```diff
@@ -129,8 +129,9 @@
 
 DEFAULT_JINJA_STRUCTURED_ELEMENTS_NAMES = [
     ("autoescape", "endautoescape"),
-    ("block", "endblock"),
+    ("blocktranslate", "plural", "endblocktranslate"),
     ("blocktrans", "plural", "endblocktrans"),
+    ("block", "endblock"),
```

Comment:
note: this re-order is required because the approach is greedy, the first match is used.

So if we want to ensure that `blocktranslate` gets pulled in and it does not get treated as `blocktrans` or `block` we need to ensure that it is first.

Hope that makes sense.

Also blocktranslate is the Django 4.0 variant.